MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -842,9 +842,12 @@ private void AppendPlaylistCodecsField(StringBuilder builder, StreamState state)
 
             StringBuilder codecs = new StringBuilder();
 
-            codecs.Append(videoCodecs)
-                .Append(',')
-                .Append(audioCodecs);
+            codecs.Append(videoCodecs);
+            if (string.IsNullOrEmpty(videoCodecs) && string.IsNullOrEmpty(audioCodecs))
```

Comment:
This seems incorrect

Reply:
Ah shoot you're right. I blame no morning coffee 😅